project/taskweaver_config.json

```diff
@@ -1,5 +1,6 @@
 {
-"llm.api_base": "https://api.openai.com/v1",
-"llm.api_key": "",
-"llm.model": "gpt-4-1106-preview"
+  "llm.api_base": "https://console.groq.com/",
```

Comment:
Could you remove the config information?

Reply:
Yes sure. I added this just as a reference. let me just remove the information from here and change it back to the original one.

Reply:
I just changed the config file and updated the branch. Let me know if any further changes needed from my side.